### PR TITLE
translation es

### PR DESCRIPTION
**Pull request by EnrikeGonzaloLegarra**
# Goal / Issue

_Describe a short description of of this PR._

Resolves issue #47 

### Quality

* [ ] Submitted a ticket for my issue if one did not already exist
* [ ] Ticket number is in the subject of the commit message
* [ ] Used [Github auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message or PR
* [ ] Updated documentation for changes to the API
* [ ] Added new tests for new functionality, and tests are passing
* [ ] Code, writing, and design changes follow the [Style guides](https://github.com/mirai-audio/mir/wiki/Style-Guides)
* [ ] Single commit - `git rebase -i` off `master`
* [ ] No dangling orphan code

_If there’s a checkbox you can’t complete for any reason, that's okay, just
explain in detail why you weren’t able to do so._

### CODE REVIEW

File: app/locales/es/about.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* eslint-disable quotes, quote-props */
+
+export default {
+    'copy': 'La aplicación de audio para unir todos los servicios de transmisión de música. Reproducir canciones ' +
```
Comment (0xadada) [inline app/locales/es/about.js]: Could you change the indentation from 4-spaces to 2-spaces?

File: app/locales/es/app.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable quotes, quote-props */
+
+export default {
+    'network': {
```
Comment (0xadada) [inline app/locales/es/app.js]: Could you change the indentation from 4-spaces to 2-spaces?

File: app/locales/es/app.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        'online': 'Con conexión',
+        'offline': 'Sin conexión',
+        'offline-detail': 'No conectado',
+    },
+
+    'noscript': {
+        'title': 'mirai.audio se utiliza mejor con JavaScript',
+        'copy': 'El euqipo de mirai.audio respeta la decision de deshabilitar JavaScript. ' +
+            'La funcionalidad de Play/Pause y sin conexión necesitan Javascript. ' +
```
Comment (0xadada) [inline app/locales/es/app.js]: Could you change the indentation from 4-spaces to 2-spaces?

**Comment by 0xadada** [comment]
Which dialect of Spanish is this? Spain-Spanish, or another country?

File: app/locales/es/about.js (hunk 1 above)
Comment (EnrikeGonzaloLegarra) [reply app/locales/es/about.js]: ok I change de indentation..this spanish is for spain.

**Comment by EnrikeGonzaloLegarra** [comment]
I have already made the changes that you have told me.
I hope they are well

**Comment by 0xadada** [comment]
Congrats on the PR, looks beautiful 🎉 💃 

screen shot 2017-10-29 at 5 06 11 pm

**Pull request closed without merging**